src/Utilities/TypeDisplayer.hpp

```diff
@@ -22,6 +24,74 @@
  *
  * \note The TypeDisplayer header should only be included during testing
  * and debugging.
+ *
+ * \see make_list
  */
 template <typename...>
 struct TypeDisplayer;
+
+/*!
+ * \ingroup Utilities
+ * \brief Metafunction to turn a parameter pack into a typelist
+ *
+ * This metafunction is really only useful for debugging metaprograms. For
+ * example, the desired algorithm might be:
+ *
+ * \code
+ * using variables_tags_from_single_tags = tmpl::filter<
+ *     extracted_from_variables,
+ *     tmpl::bind<tmpl::found, tmpl::pin<mutated_tags_list>,
+ *                tmpl::bind<std::is_same, tmpl::_1, tmpl::parent<tmpl::_1>>>>;
+ * \endcode
+ *
+ * However, getting the `tmpl::pin`, `tmpl::parent`, and `tmpl::bind` calls
+ * right can be extremely frustrating with little help as to what is going on.
+ * Let's introduce an error by pinning `tmpl::_1`:
+ *
+ * \code
+ * using variables_tags_from_single_tags = tmpl::filter<
+ *     extracted_from_variables,
+ *     tmpl::bind<tmpl::found, tmpl::pin<mutated_tags_list>,
+ *                tmpl::bind<std::is_same, tmpl::pin<tmpl::_1>,
+ *                tmpl::parent<tmpl::_1>>>>;
+ * \endcode
+ *
+ * The result is comparing all values in `extracted_from_variables` to
+ * themselves. To find this out, replace `tmpl::filter` and `tmpl::found` with
+ * `tmpl::transform`, and the metafunction `std::is_same` to `tmpl::make_list`.
+ * You will then get back a "backtrace" of what the algorithm did, which is
+ * invaluable for getting the `tmpl::pin` and `tmpl::parent` right. That is,
+ *
+ * \code
+ * using variables_tags_from_single_tags2 = tmpl::transform<
+ *     extracted_from_variables,
+ *     tmpl::bind<tmpl::transform, tmpl::pin<mutated_tags_list>,
+ *                tmpl::bind<make_list, tmpl::_1, tmpl::parent<tmpl::_1>>>>;
```

Comment:
previous text says to use `tmpl::make_list`

Reply:
Thanks! Fixed!